package.json

```diff
@@ -38,11 +38,11 @@
   "dependencies": {
     "@plugnet/util": "^1.1.100",
     "@plugnet/wasm-crypto-js": "^0.11.102",
-    "@polkadot/api": "^0.98.0-beta.0",
-    "@polkadot/reactnative-identicon": "^0.46.1",
-    "@polkadot/types": "^0.98.0-beta.1",
-    "@polkadot/util": "^1.7.1",
-    "@polkadot/util-crypto": "1.7.1",
+    "@polkadot/api": "^0.100.1",
```

Comment:
i think it's `1.0.0-beta.7`

it seems `yarn upgrade-interactive --latest` ignores the beta tags

Reply:
https://github.com/polkadot-js/api/blob/be258a1f7c997ad8e31e8b515833c943672bfb76/package.json#L2